test-outofproc/GetProductColumnTypesSerializationAsyncEnumerable.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Azure.Functions.Worker.Extension.Sql;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using DotnetIsolatedTests.Common;
+using System.Globalization;
+using System.Threading.Tasks;
+
+namespace DotnetIsolatedTests
+{
+    public static class GetProductsColumnTypesSerializationAsyncEnumerable
+    {
+        /// <summary>
+        /// This function verifies that serializing an item with various data types
+        /// and different languages works when using IAsyncEnumerable.
+        /// </summary>
+        [Function(nameof(GetProductsColumnTypesSerializationAsyncEnumerable))]
+        public static async Task<List<ProductColumnTypes>> Run(
```

Comment:
It looks like this is only fixing the Output binding tests, please remove any functions that aren't currently being used (such as this one) or update the tests to run against these ones too.

Reply:
Oh I missed your comment in the description. I guess this is fine - but in the future it'd be best to just avoid checking in code we know isn't working correctly (even if it's just test code) and instead just keeping a feature branch or something that you can use when working on fixing it. So if you want to do that now go ahead, but it's ok if you leave this in here for now then too (assuming you'll be working on fixing those right away)